lib/gruf/server.rb

```diff
@@ -31,10 +31,10 @@ class ServerAlreadyStartedError < StandardError; end
     ##
     # Initialize the server and load and setup the services
     #
-    # @param [Hash] options
+    # @param [Hash] opts
     #
-    def initialize(options = {})
-      @options = options || {}
+    def initialize(opts = {})
+      @options = opts || {}
```

Comment:
If you're going to change this name here, can you change to `@options` or `opts` on L46 below